java-frontend/src/test/java/org/sonar/java/bytecode/se/BytecodeEGWalkerTest.java

```diff
@@ -172,6 +188,36 @@ public void test_method_throwing_exception() throws Exception {
     assertThat(methodYield).isInstanceOf(ExceptionalYield.class);
   }
 
+  @Test
+  public void test_starting_states() throws Exception {
+    BytecodeEGWalker walker = new BytecodeEGWalker(new BehaviorCache(null));
+    Symbol.MethodSymbol methodSymbol = mock(JavaSymbol.MethodJavaSymbol.class);
```

Comment:
I think mock is unecessary here : new MethodJavaSymbol with new MethodJavaType should be sufficient and clearer.